crates/attestation/Cargo.toml

```diff
@@ -8,8 +8,8 @@ anyhow = { workspace = true }
 borsh = { workspace = true }
 dcap-qvl = { workspace = true }
 derive_more = { workspace = true }
-dstack-sdk-types = { workspace = true }
 hex = { workspace = true }
+interfaces = { workspace = true }
```

Comment:
Why is the attestation crate aware of our interface types? I'd like to keep this crate purely focused on attestation data structures and verification logic. It should not be concerned about our smart contract interface.